static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -21,18 +22,27 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
   const {title, subtitle} = getTitleSubtitle(event);
   const avatarSize = parseInt(space(4), 10);
 
-  // If any of data fails to load, we don't want to render the component
-  // Only "One other issue appears in the same trace. View Full Trace (X issues)" would show up
+  const referrer = 'issue_details.related_trace_issue';
+
+  trackAnalytics(`${referrer}.viewed`, {
```

Comment:
This is a side effect, so it should be inside a `useEffect` callback. But it could be better to add this to the `issue_details.viewed` event anyway